test-cypress/steps/operations-statuses-component-steps.js

```diff
@@ -0,0 +1,28 @@
+export class OperationsStatusesComponentSteps {
+
+    static getOperationsStatusesComponent() {
+        return cy.get('.operations-statuses');
+    }
+
+    static getOperationStatuses() {
+        return cy.get('.operation-status-content');
+    }
+
+    static openOperationStatusesDialog() {
+        OperationsStatusesComponentSteps.getOperationsStatusesComponent().click();
+    }
+
+    static clickOnOperationStatus(operationIndex = 0) {
+        OperationsStatusesComponentSteps.getOperationStatuses().eq(operationIndex).then(($operationElement) => {
+            expect($operationElement).to.have.attr('target', '_blank');
+            // update attr to open in same tab
+            $operationElement.attr('target', '_self');
```

Comment:
I believe that this is not necessarry. You can just verify the href and target attributes and that's it.

Reply:
Refactored